### PR TITLE
feat: add workflows for resync v1 plan/apply

### PR DESCRIPTION
**Pull request by nodegard**
## Description
Please describe the change you have made.

## Checklist:
- [ ] Tests added/updated.
- [ ] Documentation updated.
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/power-ops-sdk/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [pyproject.toml](https://github.com/cognitedata/power-ops-sdk/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: .github/workflows/resync-plan.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: Resync Plan "v1"
+
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (eriklien) [inline .github/workflows/resync-plan.yaml:6]: This means when opening a pull request for merging into main? Will it also run with new commits pushed to the branch?

**Review by eriklien — APPROVED** [review]
Looks good to me (though I can't tell if it would work).
Added a comment just for my understanding.

File: .github/workflows/resync-plan.yaml (hunk 1 above)
Comment (nodegard) [reply .github/workflows/resync-plan.yaml:6]: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

**Review by Juliamg — APPROVED** [review]
lgtm!

**Pull request merged**